.hermes/plans/2026-05-15-stale-docs-cleanup.md

```
@@ -0,0 +1,500 @@
+```md
+### Compiler Selection Caveat
+
+For reproducible local results, pass `CC=clang` explicitly:
+
+```sh
+make test CC=clang
+make smoke CC=clang
+```
+
+The Makefile uses `CC ?= clang`, but GNU Make defines a built-in `CC=cc`, so some local invocations resolve to `cc` unless the compiler is overridden on the command line.
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)